packages/ee/lib/service.ts

```diff
@@ -52,7 +63,11 @@ export const getIsEnterpriseEdition = async (): Promise<boolean> => {
     const previousResult = await getPreviousResult();
     if (previousResult.lastChecked.getTime() === new Date(0).getTime()) {
       // first call
-      await setPreviousResult({ active: true, lastChecked: new Date() });
+      await setPreviousResult({
+        active: true,
+        features: { isMultiOrgEnabled: true },
```

Comment:
I think the default value for the `isMultiOrgEnabled` for the first check should be `false`, please change this 😊

Reply:
As discussed